### DISCUSSION TITLE
Fossify Store

### DISCUSSION THREAD

**Discussion by ghost** [FossifyOrg/General-Discussion#252] [Ideas]
I would love if Fossify would have an own Fossify Store App, where we could get all of our Apps. Like for example the GrapheneOS Store. You could fork it and use it for Fossify Apps. This way it would be easier for us as Users.

**Comment by naveensingh** [comment]
I don't think Fossify apps need a dedicated store. They are easily accessible on other stores:

- https://search.f-droid.org/?q=fossify
- https://play.google.com/store/apps/dev?id=7297838378654322558

**Reply by ghost** [reply]
I understand that, of course. But was just an nice Idea.
